.github/workflows/run.yml

```diff
@@ -0,0 +1,40 @@
+name: Run Android Emulator
+
+on:
+  pull_request:
+  push:
+
+jobs:
```

Comment:
TODO: this just ignores the return codes of the commands so all fails are hidden.

Reply:
I'll fix this in a new PR.